app/javascript/app/application.js

```diff
@@ -1,17 +0,0 @@
-require([
-  'util/translation/google-translate',
-  'domReady!'
```

Comment:
I'm trying to remember why this was added. Here are the relevant commits: https://github.com/codeforamerica/ohana-web-search/commit/53e3cf2e9718ea34b526819a32feacb5be4ca3b4 https://github.com/codeforamerica/ohana-web-search/commit/4f8923c7dceda4a5d2f05c1578e516da2b2556b0

I'm pretty sure this is because the map has a large and small view, that are dynamically updated. So the infobox script, for instance, may look for the map DOM before it's available. It's possible with this removed there may be scenarios where the script throws an error. You'd want to try reloading and resizing the map a bunch and check for JS errors.

Reply:
Or it may not be applicable with your change noted in https://github.com/codeforamerica/ohana-web-search/pull/866#issuecomment-368249807

Reply:
Seems to work fine after reloading a bunch of times and resizing and zooming the map in and out.